hw6.py

```diff
@@ -0,0 +1,19 @@
+#3.
+message1 = "Have an awesome day!"
+message2 = "Have a good day!"
+message3 = "Have a wonderful day!"
+b = 10
+c = int(input("Whats's the number?"))
+if c < b:
+    print(message1)
+elif c == b:
+    print(message3)
+else:
+    print(message2)
+
```

Comment:
It would be nice to implement this exercise using while .. type a value while a condition is met.